### PR TITLE
 Fix: Multiple queries were fired

### PR DESCRIPTION
**Pull request by Koustavd18**
### Issue 
Multiple Queries were being fired and Log table time range is incorrect 
### Addressing the footer multiple query call in this PR

**Explanation**
If a user opens console and halts in the home page for a while the time range passed for fetch functions are incorrect and the Count fetch functions is fired twice for due to the duplication of timeRange object

**Solution**
The cleaning of the log store is done by a useEffect callback which executes the function after the component is rendered. This is changed so that the cleaning of the logsStore is done before the query is fired

### CODE REVIEW

**Review by balaji-jr — CHANGES_REQUESTED** [review]
Mentioned issue is still reproducible

**Comment by Koustavd18** [comment]
Time Range duplication and multiple query for footer addressed. Mutiple query and time range duplication and for event Graph will be solved in a future PR

File: src/pages/Stream/Views/Explore/useLogsFetcher.ts [hunk 1]
```diff
@@ -1,32 +1,33 @@
...
-const { setCleanStoreForStreamChange } = logsStoreReducers;
+const { setCleanLogStoreForStreamChange } = logsStoreReducers;
 
 const useLogsFetcher = (props: { schemaLoading: boolean }) => {
 	const { schemaLoading } = props;
 	const [currentStream] = useAppStore((store) => store.currentStream);
-	const [{ tableOpts, timeRange }, setLogsStore] = useLogsStore((store) => store);
+	const [{ tableOpts }, setLogsStore] = useLogsStore((store) => store);
+
+	useMemo(() => {
```
Comment (balaji-jr) [inline src/pages/Stream/Views/Explore/useLogsFetcher.ts:14]: Redundant use of useMemo here

File: .gitignore [hunk 2]
```diff
@@ -51,3 +51,4 @@ build.zip
 /coverage
 
 
+.vercel
```
Comment (balaji-jr) [inline .gitignore:54]: We have a private deployment on Vercel for our convenience, making internal testing easier.
So, as discussed earlier rmv this line.

**Review by balaji-jr — CHANGES_REQUESTED** [review]

**Pull request closed without merging**